components/ide/jetbrains/launcher/main.go

```diff
@@ -456,6 +456,7 @@ func run(launchCtx *LaunchContext) {
 
 	cmd := remoteDevServerCmd(args, launchCtx)
 	cmd.Env = append(cmd.Env, "JETBRAINS_GITPOD_BACKEND_KIND="+launchCtx.alias)
+	cmd.Env = append(cmd.Env, "JETBRAINS_RUNNING_WARMUP="+strconv.FormatBool(launchCtx.warmup))
```

Comment:
(nit) I would try to put `GITPOD` in all custom env vars to be specific that it belongs to us, i.e. `JETBRAINS_GITPOD_WARMUP`

Reply:
Updated.